src/Robo/Commands/Tests/BehatCommand.php

```diff
@@ -126,6 +126,10 @@ public function behatDefinitions($options = ['mode' => 'l']) {
 
   /**
    * Launches PHP's internal web server via `drush run-server`.
+   *
+   * @command tests:run-server
+   * @validateInsideVm
+   * @executeInDrupalVm
    */
   protected function launchWebServer() {
     if ($this->getConfigValue('behat.run-server')) {
```

Comment:
This command should probably run regardless of this config value. We'll need to move this check up a method.